### PR TITLE
Add Compose for Intellij usage example

### PR DESCRIPTION
**Pull request by AlexeyTsvetkov**

### CODE REVIEW

File: examples/intellij-plugin-with-experimental-shared-base/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+## Example CFD based plugin for IntelliJ Idea.
```
Comment (olonho) [inline examples/intellij-plugin-with-experimental-shared-base/README.md:1]: CFD -> Compose Multiplatform

File: examples/intellij-plugin-with-experimental-shared-base/settings.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+rootProject.name = "ComposeDemoPlugin"
+
+pluginManagement {
+    plugins {
+        kotlin("jvm").version(extra["kotlin.version"] as String)
+        id("org.jetbrains.compose").version(extra["compose.version"] as String)
```
Comment (olonho) [inline examples/intellij-plugin-with-experimental-shared-base/settings.gradle.kts:6]: Nice

File: examples/intellij-plugin-with-experimental-shared-base/gradle.properties [hunk 3]
```diff
@@ -0,0 +1,5 @@
+kotlin.stdlib.default.dependency=false
```
Comment (olonho) [inline examples/intellij-plugin-with-experimental-shared-base/gradle.properties:1]: Is this change required?

Comment (AlexeyTsvetkov) [reply examples/intellij-plugin-with-experimental-shared-base/gradle.properties:1]: Yes, otherwise the plugin blows up trying to load a class from Kotlin stdlib, which is already loaded by bootstrap classloader

**Review by olonho — APPROVED** [review]
